public/navigation.json

```diff
@@ -1899,6 +1899,13 @@
                   "origin": "",
                   "type": "markdown",
                   "children": []
+                },
+                {
+                  "name": "How to configure tradeName to use in the organization search field",
+                  "slug": "configure-tradename",
```

Comment:
```suggestion
                  "slug": "how-to-configure-tradename-to-use-in-the-organization-search-field",
```

Reply:
https://github.com/vtexdocs/dev-portal-content?tab=readme-ov-file#how-can-i-deal-with-page-slugs

"New guide or release notes: an article's file name will become the page's slug always. Do not create a slug that is different from the file name, the portal will not interpret it otherwise."